src/appengine/handlers/jobs.py

```diff
@@ -123,6 +140,39 @@ def post(self):
           'Job name can only contain letters, numbers, dashes and underscores.',
           400)
 
+    # Retriving old associated fuzzers.
+    query = data_types.Fuzzer.query()
+    query = query.filter(data_types.Fuzzer.jobs == name)
+    old_fuzzers = ndb_utils.get_all_from_query(query)
```

Comment:
nit: don't use get_all_from_query in new code. It's not longer needed and will be removed at some point.

Reply:
done